### ISSUE TITLE
Understanding Version reporting to SmartThings hub

### ISSUE THREAD

**Issue by bluesubaru03** [bwssytems/ha-bridge#1163]
How is the HA-Bridge reporting it's version to devices trying to discover it? My Smartthings Hub (V2) is always trying to communicate to the bridge via port 80 even though the bridge is setup on port 443. I have had ST support manually update the port in my IDE to 443 and the integration works fine for awhile until something sets it back to 80. Looking at the description.xml file shows it is reporting the proper port 443. In issue #1107, ST developer posborne states that the ST hub is looking at the version the bridge is reporting and setting the port based on "uses the version of the bridge (1.24+ for TLS) as a marker for TLS support or, temporarily, just using port 80 as we have in the past."  I also know there is a also firmware version setting on the bridge. Where/how is this reported to ST hub?  In the description.xml file there is a version section that is reporting 1.0, is this were it is providfing the version to ST? Can the bridge version be modified?

**Comment by audiofreak9** [comment]
Yes, you can update the version HA Bridge is reporting on the Bridge Control page per the Wiki here: [https://github.com/bwssytems/ha-bridge/wiki/HA-Bridge-FAQs#huereleaseversion](https://github.com/bwssytems/ha-bridge/wiki/HA-Bridge-FAQs#huereleaseversion)

**Comment by bluesubaru03** [comment]
According to SmartThings staff the Hub is looking for APIVERSION to determine the port to use. "The SmartThings hub does not inspect the description.xml, but looks at the Hue API version that the bridge reports in a HTTP config response. If apiversion is greater of equal to 1.24, it will use TLS on 443. If the the version is less than 1.24 or the Hue bridge does not report apiversion, it will default to HTTP on port 80." 
Is this reported by HA-Bridge?

**Comment by bwssytems** [comment]
The Api version is set in the HueConstants.java and is 1.19. This would need to be updated in code to change it.

**Comment by bluesubaru03** [comment]
Can this be requested as an enhancement? Set API to 1.24 to allow ST to communicate over 443 and set to 1.19 to allow communications over port 80?

**Comment by bwssytems** [comment]
@bluesubaru03 This can be requested as an enhancement
